### PR TITLE
Feature/plant list

### LINKED ISSUE #1: #1 Plant List
## Value proposition
As a user
I want to browse a list of plants,
so that I can discover new plants and learn about their care requirements.
## Description

Image

## Acceptance criteria

- [ ] The homepage displays a list of plants.
- [ ] The plant list is headlined with its purpose.
- [ ] Each plant listing includes:
   - [ ] Plant name
   - [ ] Plant botanical name
   - [ ] Plant image
- [ ] The list supports vertical scrolling to accommodate multiple entries.

## Tasks

- [ ] Create feature branch feature/plant-list
- [ ]  Inspect and adapt the example data in the [assets folder](https://github.com/neuefische/hh-web-24-8/blob/main/capstone/plant-pal/assets/plants.json)
   - [ ]  Add an imageUrl for each plant, for example from [unsplash](https://unsplash.com/de)
- [ ] create component `PlantCard` to render a list in index.js
- [ ] display plant name, botanical name, plant image on each card
- [ ] create a button "see more" in each card
- [ ] map cards for each plant 
- [ ] add image clearance for host url in next.config.js

### PR DESCRIPTION
**Pull request by ana-nova**

### CODE REVIEW

File: components/PlantCard.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function PlantCard({ plant }) {
+  return (
+    <Container>
+      <h2>{plant.name}</h2>
+      <StyledParagraph>{plant.botanicalName}</StyledParagraph>
+      <RoundImage alt="image" src={plant.imageUrl}></RoundImage>
```
Comment (onemanwenttomow) [inline components/PlantCard.js]: I think we need to make the alt more descriptive for accessibility reasons. 

Something like

``` javascript
  alt={`Photo of ${plant.name}`} 
```

File: components/PlantCard.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  );
+}
+
+const RoundImage = styled.img`
+  border-radius: 100px;
+  width: 200px;
+  height: 200px;
+`;
+
+const Container = styled.div`
```
Comment (onemanwenttomow) [inline components/PlantCard.js]: Could this potentially be an `article` element? It would improve the semantics.

File: components/PlantList.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  return (
+    <div>
+      <StyledHeader>My Plant Collection</StyledHeader>
+      <StyledList>
+        {plants.map((plant) => (
+          <li key={plant.id}>
+            <PlantCard plant={plant} />
+          </li>
+        ))}
+      </StyledList>
```
Comment (onemanwenttomow) [inline components/PlantList.js:17]: What if there are no plants? If that is a scenario that could happen, then we would end up with an empty `<ul>`.

If the plants could be empty at some point, then we would probably want to do a check before displaying a `<ul>`.

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Nice work team. I have one comment on the `alt` tag that i think we should work on. I also added a couple of potential suggestions that you can implement if you wish.

File: components/PlantCard.js (hunk 1 above)
Comment (frush86) [reply components/PlantCard.js]: that makes sense. thank you

File: components/PlantCard.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function PlantCard({ plant }) {
+  return (
+    <Container>
+      <h2>{plant.name}</h2>
+      <StyledParagraph>{plant.botanicalName}</StyledParagraph>
+      <RoundImage
+        alt={`Image of ${plant.name}`}
```
Comment (onemanwenttomow) [inline components/PlantCard.js:10]: I know i suggested something similar... and this is really nitpicking now. But i guess we would just use `plant.name`

> Redundant alt attribute. Screen-readers already announce `img` tags as an image. You don’t need to use the words `image`, `photo,` or `picture` (or any specified custom words) in the alt prop.

**Review by F-Kirchhoff — APPROVED** [review]

**Review by onemanwenttomow — APPROVED** [review]
Nice job! Had one more tiny point on the alt tag, but consider this approved  🎉

**Pull request merged**